source/uwp/Renderer/lib/Util.cpp

```diff
@@ -740,4 +749,24 @@ HRESULT ProjectedElementTypeToHString(ABI::AdaptiveCards::Rendering::Uwp::Elemen
 {
     CardElementType sharedElementType = static_cast<CardElementType>(projectedElementType);
     return UTF8ToHString(CardElementTypeToString(sharedElementType), result);
+}
+
+std::wstring StringToWstring(const std::string& in) 
+{
+    std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>, wchar_t> utfConverter;
+    return utfConverter.from_bytes(in);
+}
+
+std::string WstringToString(const std::wstring& input)
+{
+    if (sizeof(wchar_t) == 2)
```

Comment:
Can you add a comment here so it's clear to future developers why we need this check?